unstable/wlr-layer-shell-unstable-v1.xml

```diff
@@ -86,11 +86,27 @@
       are designed to be rendered as a layer of a stacked desktop-like
       environment.
 
-      Layer surface state (anchor, exclusive zone, margin, interactivity) is
-      double-buffered, and will be applied at the time wl_surface.commit of the
-      corresponding wl_surface is called.
+      Layer surface state (size, anchor, exclusive zone, margin, interactivity)
+      is double-buffered, and will be applied at the time wl_surface.commit of
+      the corresponding wl_surface is called.
     </description>
 
+    <request name="set_size">
+      <description summary="sets the size of the surface">
+        Sets the size of the surface in pixels. The compositor will display the
```

Comment:
Pixels is a little ambiguous here: is it in surface-local coordinates or in buffer-local coordinates? If the surface has a scale factor of 2 and a buffer size of 42, the surface size is 21.

So we should probably s/in pixels/in surface-local coordinates/.